Core/src/main/java/put/io/black/java/core/logic/FileManager.java

```diff
@@ -0,0 +1,73 @@
+package put.io.black.java.core.logic;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class FileManager {
+
+    public static final String PATH = "Scenarios/";
+    public static final String FILE_NOT_EXIST = "File not exist";
+
+    private boolean ready;
+
+    public FileManager() {
+        new File(PATH).mkdir();
+        File file = new File(PATH);
+        ready = file.exists();
+    }
+
+    public boolean saveScenarioText(String title, String scenario) {
+        try {
+            if (fileExist(title)){
+                return false;
+            }
+
+            PrintWriter printWriter = new PrintWriter(PATH + title + ".txt");
+            printWriter.write(scenario);
+            printWriter.flush();
+            return true;
+        } catch (FileNotFoundException e) {
+            e.printStackTrace();
+        }
+        return false;
+    }
+
+    public String readScenario(String scenarioName) {
+        String scenario = "";
+        try {
+            if (!fileExist(scenarioName)){
+                return FILE_NOT_EXIST;
```

Comment:
I will delete this and return single String. Now you prepared field for only this message.